client/my-sites/stats/annual-highlights-section/index.tsx

```diff
@@ -34,7 +35,7 @@ const FOLLOWERS_QUERY = { type: 'wpcom', max: 0 };
 
 // Meant to replace annual-site-stats section
 export default function AnnualHighlightsSection( { siteId }: { siteId: number } ) {
-	const year = new Date().getFullYear();
+	const [ year, setYear ] = useState( new Date().getFullYear() );
```

Comment:
Could we hide the followers count when viewing a non-current year? This is what I have in mind:

#### This year:
<img width="1250" alt="image" src="https://user-images.githubusercontent.com/4044428/208997093-b9be18cf-014f-4883-bc85-9f312a37310c.png">

#### Any past years:
<img width="1255" alt="image" src="https://user-images.githubusercontent.com/4044428/208997175-f4420ef9-aa09-4c23-ac13-f17f2113e227.png">

We can probably accomplish this by tweaking the count determination on line 56:
```js
        followers: year === currentYear ? ( followers?.total_wpcom ?? null ) : null,
```

Reply:
Good point. What do you think about hiding the card (visually) instead of showing a hyphen?

Reply:
I think it'll be better to keep the followers card there to avoid reflowing the other four cards. I'll defer to our designers if they say otherwise, but blanking the count with a hyphen is probably a conservative approach here.

Reply:
Sounds good. I also updated the testing instructions.